actionview/lib/action_view/helpers/csrf_helper.rb

```diff
@@ -21,10 +21,13 @@ module CsrfHelper
       #
       def csrf_meta_tags
         if defined?(protect_against_forgery?) && protect_against_forgery?
-          [
-            tag("meta", name: "csrf-param", content: request_forgery_protection_token),
-            tag("meta", name: "csrf-token", content: form_authenticity_token)
-          ].join("\n").html_safe
+          safe_join(
+            [
+              tag("meta", name: "csrf-param", content: request_forgery_protection_token),
+              tag("meta", name: "csrf-token", content: form_authenticity_token)
+            ],
+            "\n"
+          )
```

Comment:
While we're changing this method, would it be appropriate to replace the calls to `tag("meta")` with `tag.meta`?

```suggestion
          safe_join(
            [
              tag.meta(name: "csrf-param", content: request_forgery_protection_token),
              tag.meta(name: "csrf-token", content: form_authenticity_token)
            ],
            "\n"
          )
```

Reply:
Definitely!